examples/protocol_buffers/BUILD.bazel

```diff
@@ -55,10 +55,48 @@ ts_devserver(
 )
 
 # Test for production mode
-load("@build_bazel_rules_nodejs//:defs.bzl", "rollup_bundle")
+load("@build_bazel_rules_nodejs//:defs.bzl", "rollup_bundle", "nodejs_binary")
 
 rollup_bundle(
     name = "bundle",
     deps = [":app"],
     entry_point = "examples/protocol_buffers/app",
+    globals = {
+        "long": "Long",
+        "protobufjs/minimal": "protobuf",
+    },
+)
+
+# Needed because the prodserver only loads static files that appear under this
+# package.
+genrule(
+    name = "protobufjs",
+    srcs = ["@build_bazel_rules_typescript_protobufs_compiletime_deps//:node_modules/protobufjs/dist/minimal/protobuf.min.js"],
```

Comment:
isn't there a filegroup that gives both files? can we have one genrule that uses that?

Reply:
there must be a way! but it's okay if not

Reply:
Found a way